### ISSUE TITLE
Muted text does lacks sufficient color contrast

### ISSUE THREAD

**Issue by axamei** [pulibrary/requests#350]
Violation: Elements must have sufficient color contrast.

I understand we want the text to be muted, but it must have a minimum color contrast of 4.5. `.text-muted` is currently at a color contrast of 4.07 (foreground color: #777777, background color: #f5f4f1).
